Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs

```diff
@@ -88,22 +91,22 @@ private void OnPlayersSpawning(RulePlayerSpawningEvent ev)
             ops[i] = _random.PickAndTake(ev.PlayerPool);
         }
 
-        var map = "/Maps/syndiepuddle.yml";
+        var map = "/Maps/infiltrator.yml";
 
         var aabbs = _stationSystem.Stations.SelectMany(x =>
-            Comp<StationDataComponent>(x).Grids.Select(x => _mapManager.GetGridComp(x).Grid.WorldBounds.CalcBoundingBox())).ToArray();
+            Comp<StationDataComponent>(x).Grids.Select(x => _mapManager.GetGridComp(x).Grid.WorldAABB)).ToArray();
         var aabb = aabbs[0];
         for (int i = 1; i < aabbs.Length; i++)
         {
             aabb.Union(aabbs[i]);
         }
 
-        var (_, grid) = _mapLoader.LoadBlueprint(GameTicker.DefaultMap, map, new MapLoadOptions
+        var (_, gridId) = _mapLoader.LoadBlueprint(GameTicker.DefaultMap, map, new MapLoadOptions
         {
-            Offset = aabb.Center + MathF.Max(aabb.Height, aabb.Width)*2.5f
+            Offset = aabb.Center + MathF.Max(aabb.Height / 2f, aabb.Width / 2f) * 2.5f
```

Comment:
funnily enough the missing /2 was intended but we can move the shuttle closer too, idc. the idea was to give the crew a tad more of a buffer.

Reply:
You could give the crew a buffer by disabling the shuttle for a few minutes for a "planning/equipping" phase

Reply:
I know they have pinpointer but based on the average IQ I saw I feared them getting lost.